### PR TITLE
Added Network module and placeholder in KFImage

### PR DESCRIPTION
**Pull request by KritikHub**
- Added placeholder property in MovieBacdropCard, MoviePosterCard, MovieDetailView
- Added placeholder image in Assets
- Added following files in network module- EndPoint, URN, BaseUrlType

### CODE REVIEW

**Comment by KritikHub** [comment]
@JustManish Please review the changes

File: SwiftUiMovieDb/Configuration/Configuration.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//
+//  Configuration.swift
+//  SwiftUiMovieDb
+//
+//  Created by mac on 03/11/23.
+//
+
+import Foundation
```
Comment (JustManish) [inline SwiftUiMovieDb/Configuration/Configuration.swift]: Add TODO Comments, what you will be doing in this file, if it's empty.

//TODO: Need to add Configuration

File: SwiftUiMovieDb/Network/EndPoint.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import Foundation
+
+enum EndPoint {
+    case nowPlaying
+    case upcoming
+    case topRated
+    case popular
+    
+    var description: String {
```
Comment (JustManish) [inline SwiftUiMovieDb/Network/EndPoint.swift]: Do we need API EndPoint Description, if not consider removing it.

File: SwiftUiMovieDb/Views/MovieDetailView.swift [hunk 3]
```diff
@@ -133,18 +133,18 @@ struct MovieDetailView: View {
 struct MovieDetailImage: View {
     
     let imageURL: URL
-    let cache = ImageCache.default
     
     var body: some View {
         ZStack {
             Rectangle().fill(Color.gray.opacity(0.3))
```
Comment (JustManish) [inline SwiftUiMovieDb/Views/MovieDetailView.swift:139]: .fill() should be in Next Line.

File: SwiftUiMovieDb/Views/MoviePosterCard.swift [hunk 4]
```diff
@@ -11,31 +11,33 @@ import Kingfisher
 struct MoviePosterCard: View {
     
     let movie: Movie
-    let cache = ImageCache.default
+    private let imageCornerRadius: CGFloat = 8
+    private let imageShadowRadius: CGFloat = 4
+    private let frameWidth: CGFloat = 204
```
Comment (JustManish) [inline SwiftUiMovieDb/Views/MoviePosterCard.swift:16]: Let's try to use relative height to screen width for such big dimensions.

eg: frameWidth = screenWidth * 0.7 and height = screenWidth

Let's see if this works.

**Review by JustManish — APPROVED** [review]

**Pull request closed without merging**